package.yaml

```diff
@@ -85,7 +85,6 @@ tests:
     main: Spec.hs
     source-dirs:
       - tests
-      - src
```

Comment:
I hope sensei is unaffected, because after `make sensei`, I'm seeing `test -d .stack-work/ && echo "stack exec --" || echo "cabal exec --" sensei -isrc -itests tests/Spec.hs`, note the `-isrc`. The automatic reloading and testing doesn't work for me, but this is unrelated and I don't care, I'm fine with my universal emacs setup, which doesn't overheat my laptop, until I tell it to. :)

In my setup, I'm just building with 'cabal build' (in my emacs window) and with the 'src' it compiles all the modules again for the tests target, hence the change. I guess it's doing the same in other setups, too, compiling the modules twice, instead of using their already compiled (or interpreted?) version from the lib.